ui/user.html

```diff
@@ -0,0 +1,182 @@
+				<img src="./img/rides/car1.jpeg">
+				<div class="details">
+					<p>Price : $130 <a href="">View ride details</a></p>
+					<span class="tooltip"><a href="">Mbuvi</a>
+  						<span class="tooltiptext">
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star"></span>
+						<span class="fa fa-star"></span>
+					    View driver rating and reviews by users</span>
+				    </span>
+					
+				</div>
+			  
+			</div>
+
+			<div class="col-sm-12 col-md-3">
+				<img src="./img/rides/car3.jpeg">
+				<div class="details">
+					<p>Price : $130 <a href="">View ride details</a></p>
+					<span class="tooltip"><a href="">Mbuvi</a>
+  						<span class="tooltiptext">
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star"></span>
+						<span class="fa fa-star"></span>
+					    View driver rating and reviews by users</span>
+				    </span>
+					
+				</div>
+				
+			</div>
+			<div class="col-sm-12 col-md-3">
+				<img src="./img/rides/car4.jpg">
+				<div class="details">
+					<p>Price : $130 <a href="">View ride details</a></p>
+					<span class="tooltip"><a href="">Mbuvi</a>
+  						<span class="tooltiptext">
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star"></span>
+						<span class="fa fa-star"></span>
+					    View driver rating and reviews by users</span>
+				    </span>
+					
+				</div>
+			</div>
+		</div>	
+
+		<!--second row for rides-->	
+		<div class="row rides">
+			<div class="col-sm-12 col-md-3">
+				<img src="./img/rides/car0.jpeg">
+				<div class="details">
+					<p>Price : $130 <a href="">View ride details</a></p>
+					<span class="tooltip"><a href="">Mbuvi</a>
+  						<span class="tooltiptext">
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star"></span>
+						<span class="fa fa-star"></span>
+					    View driver rating and reviews by users</span>
+				    </span>
+					
+				</div>
+				
+			</div>
+			<div class="col-sm-12 col-md-3">
+				<img src="./img/rides/car1.jpeg">
+				<div class="details">
+					<p>Price : $130 <a href="">View ride details</a></p>
+					<span class="tooltip"><a href="">Mbuvi</a>
+  						<span class="tooltiptext">
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star checked"></span>
+						<span class="fa fa-star"></span>
+						<span class="fa fa-star"></span>
+					    View driver rating and reviews by users</span>
+				    </span>
+					
+				</div>
+			  
+			</div>
+
+			<div class="col-sm-12 col-md-3">
```

Comment:
Did you go through the instructions on the project document, I reckon this is bootstrap?